### PR TITLE
Remove extraneous close parenthesis

### LINKED ISSUE #17964: Typo: extraneous close parenthesis
After the "Analyze the core dump" header, the link to the `dotnet-dump` tool is followed by an extraneous close parenthesis:

&gt; Now that you have a core dump generated, use the dotnet-dump) tool to analyze the dump:

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 2913d994-4c72-cc6e-0bd4-ba122ae21753
* Version Independent ID: 03f3761e-be31-bae9-6ac7-ad86db389212
* Content: [Debug a memory leak tutorial](https://docs.microsoft.com/en-us/dotnet/core/diagnostics/debug-memory-leak#feedback)
* Content Source: [docs/core/diagnostics/debug-memory-leak.md](https://github.com/dotnet/docs/blob/master/docs/core/diagnostics/debug-memory-leak.md)
* Product: **dotnet-core**
* GitHub Login: @sdmaclea
* Microsoft Alias: **stmaclea**

### PR DESCRIPTION
**Pull request by IEvangelist**
## Summary

Remove extraneous close parenthesis

Fixes #17964

### CODE REVIEW

**Review by tdykstra — APPROVED** [review]
LGTM

File: docs/core/diagnostics/debug-memory-leak.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Debug a memory leak tutorial
 description: Learn how to debug a memory leak in .NET Core.
 ms.topic: tutorial
-ms.date: 12/17/2019
+ms.date: 04/20/2020
```
Comment (gewarren) [inline docs/core/diagnostics/debug-memory-leak.md:5]: Does this really warrant a date update?

Comment (IEvangelist) [reply docs/core/diagnostics/debug-memory-leak.md:5]: I can revert if you'd like, typically I read through the article if I'm making updates and verify that everything looks accurate. So it was kind of like a freshness pass. Let me know if you want me to change it back.

**Comment by logiclrd** [comment]
Hey! Awesome to see such a rapid turnaround on the report. I just thought I'd make your Tuesday a little bit better by mentioning that I've noticed a second instance of the same error in the file, on line 111. :-D

**Pull request merged**